### PR TITLE
Clarity tweaks to API docs

### PR DESCRIPTION
**Pull request by chelsea**
Replace specific Sendle ID and API key in doco with more obvious placeholder

Corrected description of 500 errors

Made use of HTTP auth explicit rather than implied via the cURL `-u` usage

### CODE REVIEW

File: source/includes/_errors.md [hunk 1]
```diff
@@ -14,36 +14,27 @@ Error Code | Meaning
 [402](#402-payment-required) | **Payment Required** -- The client does not yet have a payment-method set up on their account, and cannot create orders.
 [404](#404-not-found) | **Not Found** -- The requested resource/URI was not found.
 [422](#422-unprocessable-entity) | **Unprocessable Entity** -- The server was unable to complete the request due to the data itself. For example, validations within the data may fail, or an upstream request may not be able to be fulfilled with the data. This is different to `400 Bad Request` as `422 Unprocessable Entity` suggests that the request sent by the client was structurally valid, and the request was attempted.
-[500](#500-internal-server-error) | **Internal Server Error** -- An error occurred within the system itself. This may be a coding error that caused the system to fail. The development team will have been notified of this failure.
```
Comment (mendokusai) [inline source/includes/_errors.md]: is the 500 being taken out because it doesn't need definition?

Comment (chelsea) [inline source/includes/_errors.md]: Yeah, in an ideal world a 500 won't be happening and and the error is fairly self explanatory with https://github.com/sendle/sendle/pull/1274

Comment (mendokusai) [inline source/includes/_errors.md]: Sure, my only hesitation to pulling it out is for total non-technicals trying to use Sendle API, but even then a 500 is pretty easy to find anywhere in the tubes... :+1:

**Comment by mendokusai** [comment]
:rocket: :guitar:

**Pull request merged**